### PR TITLE
https://issues.redhat.com/browse/ACM-7071 Remove API 2.9

### PR DESCRIPTION
**Pull request by oafischer**
Should we remove the deprecated entries for the 2 APIs as well?

### CODE REVIEW

**Review by cadawson — CHANGES_REQUESTED** [review]
I think we should remove the deprecation when it moves to the removed state. Other than that, it looks good. I don't need to review again. Thanks!

File: release_notes/deprecate_remove.adoc [hunk 1]
```diff
@@ -73,6 +61,18 @@ Learn when parts of the product are deprecated or removed from {product-title}.
 |===
 | Product or category | Affected item | Version | Recommended action | More details and links
 
+| ManagedClusterSets
+| The `v1beta1` API is removed.
+| 2.9
+| Use `v1beta2` instead.
+| ManagedClusterSets.cluster.open-cluster-management.io
```
Comment (dockerymick) [inline release_notes/deprecate_remove.adoc]:
```suggestion
| `ManagedClusterSets.cluster.open-cluster-management.io`
```

File: release_notes/deprecate_remove.adoc [hunk 2]
```diff
@@ -73,6 +61,18 @@ Learn when parts of the product are deprecated or removed from {product-title}.
...
+
+| ManagedClusterSetBindings
+| The `v1beta1` API is removed.
+| 2.9
+| Use `v1beta2` instead.
+| ManagedClusterSetBindings.cluster.open-cluster-management.io
```
Comment (dockerymick) [inline release_notes/deprecate_remove.adoc]:
```suggestion
| `ManagedClusterSetBindings.cluster.open-cluster-management.io`
```

**Pull request merged**